src/components/timeline/LayerActivity.svelte

```diff
@@ -98,10 +99,8 @@
     if (dragOffsetX && dragPoint) {
       const x = offsetX - dragOffsetX;
       const unixEpochTime = xScaleView.invert(x).getTime();
-      const start_time_doy = getDoyTime(new Date(unixEpochTime));
-      if (unixEpochTime !== dragPoint.x) {
-        effects.updateActivityDirective(dragPoint.id, { start_time_doy }, false);
-      }
+      dragPoint.x = unixEpochTime;
```

Comment:
One small side effect of this change is that there seems to be less re-arranging of overlapping items on the timeline during drag. Some amount of re-arranging is happening but it's reduced and you can sometimes drag items on top of each other, though when you stop dragging the items will snap to to the right location.

Reply:
Yeah I think this is okay for now since it is much more performant. Before calling `effects.updateActivityDirective` on each drag was causing the entire `activitiesMap` to be recalculated which is wasteful. I will remedy this in the future tho when we use streaming subscriptions so we can update an `activitiesMap` value without re-computing the entire thing.

Reply:
I'll add a TODO in the meantime there

Reply:
After this is merged I'll do a follow up PR that brings point generation into each component so we make sure points are sorted before drawing, which should fix this issue in the short term.